demo/tlschannel.js

```diff
@@ -0,0 +1,101 @@
+
+
+// A wrapper that combines a WebSocket to the channelserver
+// with a TLS Connection for "encryption".
+//
+// This is mainly to help us test out the API and data flow
+// on something approaching real production usage patterns.
+
+const CHANNEL_SERVER = 'wss://dev.channelserver.nonprod.cloudops.mozgcp.net/v1/ws/'
+
+const FAKE_PSK = tls.utf8ToBytes('fake psk');
+const FAKE_PSK_ID = tls.utf8ToBytes('fake psk id');
+
+class InsecureTLSChannel {
+
+    constructor() {
+        this.socket = null
```

Comment:
no semicolons in this? I think we should >_>

Reply:
The auth-server has ruined my ability to be consistent with semicolons; we should probably add in eslint here to keep us on track.